nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/nf-error-handler.js

```diff
@@ -89,7 +91,7 @@
             }
 
             // status code 400, 404, and 409 are expected response codes for nfCommon errors.
```

Comment:
This comment should be updated to include 413.

Reply:
Comment updated.